tests/simulators/carla/test_carla.py

```diff
@@ -10,6 +10,27 @@ def test_basic(loadLocalScenario):
     scenario = loadLocalScenario('basic.scenic')
     scenario.generate(maxIterations=1000)
 
+
+def test_simulator_import():
+    pytest.importorskip("carla")
+    from scenic.simulators.carla import CarlaSimulator
+
+
+def test_consistent_object_type():
+    pytest.importorskip("carla")
+    mapPath = "../../formats/opendrive/maps/CARLA/Town01.xodr"
+    code = f"""
+        param map = r'{mapPath}'
+        param carla_map = 'Town01'
+        model scenic.simulators.carla.model
+        action = SetGearAction(0)
+        ego = Car
+        assert action.canBeTakenBy(ego)
+    """
+    for _ in range(2):
+        compileScenic(code, mode2D=True)
```

Comment:
Hmm, how did this work under 2.x? There shouldn't be any `mode2D` option. Did you try testing with CARLA installed?

Reply:
Good catch, fixed.